src/Compilers/Core/Portable/CodeAnalysisEventSource.cs

```diff
@@ -95,6 +96,12 @@ internal unsafe void NodeTransform(int nodeHashCode, string name, string tableTy
             }
         }
 
+        [Event(7, Message = "Compilation {0} started", Keywords = Keywords.Performance, Level = EventLevel.Informational, Opcode = EventOpcode.Start, Task = Tasks.Compilation)]
```

Comment:
Do we want to disambiguate that this is a CompilerServer Compilation?

Reply:
Not sure if you're suggesting the message should be diff or I should use different names for the methods here.

Reply:
I would probably use different event names, just so its super obvious from an ETW trace what it represents. We don't have it for non-server compilations right now, but there seems like a future where we might?